sydent/sydent.py

```diff
@@ -77,6 +77,7 @@
         'replication.https.bind_address': '::',
         'replication.https.port': '4434',
         'obey_x_forwarded_for': 'False',
+        'federation.verifycerts': 'False',
```

Comment:
I think there's a strong argument for setting this to True by default now; I'd prefer not to have to remember to revisit this. I don't think we have anyone pulling sydent updates regularly, and we're already at 60% and rising. Anyone who does update sydent (including matrix.org/vector.im/dinsic) can use the config setting to disable the check until they are ready to enforce it.

Reply:
Got it, setting to True then :+1: